pages/your_domain/rare_disease_data.md

```diff
@@ -5,6 +5,7 @@ contributors: [Philip van Damme, Nirupama Benis, César Bernabé, Shuxin Zhang,
 page_id: rare_disease
 related_pages: 
   your_domain: [human_data]
+  your_role: [it_support, data_manager]
```

Comment:
@floradanna I don't think this is allowed?

Reply:
It is not allowed, indeed: https://rdmkit.elixir-europe.org/editorial_board_guide#related-pages